### ISSUE TITLE
Setting iframeHost breaks link

### ISSUE THREAD

**Issue by varnent** [fightforthefuture/digital-climate-strike#95]
Links all break when we set iframeHost.

`iframeHost: '/wp-content/themes/shiro/assets/digitalclimatestrike',`

Anyone else had similar issue? Any known fix?

**Comment by drumm** [comment]
Yes, I ran into this too. I had to hack widget.js around this. A couple potential fixes are at https://github.com/fightforthefuture/digital-climate-strike/issues/90#issuecomment-533158647

**Comment by varnent** [comment]
Thank you @drumm for the tip! Still running into problems: https://wikimediafoundation-org-develop.go-vip.co/

**Comment by drumm** [comment]
Our complete fix, that worked for us, was setting `iframeHost: 'https://example.org'` along with hacking the path into `widget.js`, https://git.drupalcode.org/project/drupalorg/blob/0edc477f83ed56b0b6d1a30f6e2ef3d1e0e0dd41/drupalorg/digital-climate-strike/widget.js#L22

It seems like the iframe’s click events are passed back to the host document’s `widget.js`. Those events’ origin is validated at https://github.com/fightforthefuture/digital-climate-strike/blob/master/static/widget.js#L149, which only looks at the URL scheme and hostname.

Self-hosting in a subdirectory wasn’t really anticipated.

**Comment by varnent** [comment]
That did it - thank you @drumm!  :)
